src/openlifu/photoscan.py

```diff
@@ -207,3 +210,48 @@ def convert_between_ras_and_lps(mesh : vtk.vtkPointSet) -> vtk.vtkPointSet:
     transformFilter.Update()
 
     return transformFilter.GetOutput()
+
+def run_reconstruction(images: list[Path],
+                       pipeline: Path = Path(__file__).resolve().parent / "meshroom_pipelines" / "default_pipeline.mg") -> Photoscan:
```

Comment:
Can we
- make `pipeline` default to `None` and have type `Optional[Path]`. Then inside the function you see if it's `None` and set it to the right thing if so.
- instead of messing with `__file__` here, use [importlib](https://docs.python.org/3/library/importlib.resources.html#importlib.resources.path): `importlib.resources.path("openlifu.meshroom_pipelines", "default_pipeline.mg")` to get the path